extension/src/init.rs

```diff
@@ -198,15 +198,27 @@ pub fn get_column_datatype(schema: &str, table: &str, column: &str) -> Result<St
             table_schema = $1
             AND table_name = $2
             AND column_name = $3    
-    ",
+        ",
         vec![
             (PgBuiltInOids::TEXTOID.oid(), schema.into_datum()),
             (PgBuiltInOids::TEXTOID.oid(), table.into_datum()),
             (PgBuiltInOids::TEXTOID.oid(), column.into_datum()),
         ],
-    )?
-    .context(format!(
-        "could not determine data type of column `{column}` on relation: `{schema}.{table}`"
-    ))
-    .context("no resultset for column datatype")
+    )
+    .map_err(|_| {
+        anyhow!(
+            "One of schema:`{}`, table:`{}`, column:`{}` does not exist.",
+            schema,
+            table,
+            column
+        )
+    })?
+    .ok_or_else(|| {
+        anyhow!(
+            "Table `{}.{}` is empty, does not exist, or column `{}` does not exist.",
```

Comment:
I wasn't able to reproduce this error. It succeeds when the table is empty, and the error above is returned when the column or table do not exist.

Reply:
I think in this case it might be better to say "unknown error" if we truly dont know why this would be `None`. But seems like the table name, column , or presence of data in the table is definitely not the problem, so could be confusing to indicate that it is one of them.